pkg/controller/podautoscaler/replica_calculator.go

```diff
@@ -210,7 +211,7 @@ func (c *ReplicaCalculator) calcPlainMetricReplicas(metrics metricsclient.PodMet
 
 	if len(missingPods) > 0 {
 		if usageRatio < 1.0 {
-			// on a scale-down, treat missing pods as using 100% of the resource request
+			// on a scale-down, treat missing pods as using 100% (all) of the resource request
```

Comment:
let's reword to: "(...) as using exactly the target amount" while you're touching this comment. This reference to 100% of resource request doesn't make sense for `AverageValue` metric type here, seems like a bad copy&paste from `GetResourceReplicas `.

Reply:
Addressed